README.md

```diff
@@ -111,13 +111,28 @@ You should see all tests passing.
 
 Note: the tests are broken up into separate files because they rely on different tensorflow execution states (i.e. the bc tests run tf in eager mode, while rllib requires tensorflow to be running symbollically). Going forward, it would probably be best to standardize the tensorflow execution state, or re-write the code such that it is robust to execution state.
 
+## Training examples
+
+You can also try to train agents for 5 classical layouts
+
+```bash
+(harl_rllib) $ cd ppo
+(harl_rllib) ppo $ bash run_experiments.sh
+```
+
+Please select the correct `num_workers` parameter, otherwise the training of the agents wouldn't be able to start.
```

Comment:
Maybe rephrase to ~"Choose the `num_workers` parameters to be less than the number of CPUs on your machine." (is that right?)

Reply:
Right, that's how I understood it from training. And simply saying yes. But I've also added additional link to the ray documentation